lib/views/home.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import '../widget/carousels.dart';
+import '../models/movie_category.dart';
+import '../utils/general_constants.dart';
+import '../models/movie.dart';
+import '../models/genre.dart';
+import '../models/repository/movie_repository.dart';
+import '../models/repository/genre_repository.dart';
+
+class Home extends StatefulWidget {
+  const Home({
+    super.key,
+  });
+
+  @override
+  State<Home> createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  static const int positionMovies = 0;
+  static const int positionGenres = 1;
+  static const String error = 'Error:';
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: Future.wait(
+          [MovieRepository().getData(), GenreRepository().getData()]),
+      builder: (context, AsyncSnapshot<List<dynamic>> snapshot) {
```

Comment:
Add the type of context, and also you missed a trailing comma

Reply:
done